### PR TITLE
add missing packages for the prehook commit script

### PR DESCRIPTION
**Pull request by Gustry**
## Description

Add some missing packages for the prehook commit script

I'm not sure how to update the main INSTALL file.

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `fixes #11111` in the commit message next to the description
- [ ] Commits which add new features are tagged with `[FEATURE]` in the commit message
- [ ] Commits which change the UI or existing user workflows are tagged with `[needs-docs]` in the commit message and contain sufficient information in the commit message to be documented
- [ ] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTE.md#contributing-to-qgis) before each commit

### CODE REVIEW

File: doc/linux.t2t [hunk 1]
```diff
@@ -59,6 +59,16 @@ See http://qgis.org/en/site/forusers/alldownloads.html#debian-ubuntu for
...
 and we build with GDAL2 from ubuntugis).
 
+If you plan to code, you will need these packages:
+```
+apt install flip python-autopep8
+```
+
+And also a perl library:
+```
+sudo cpan YAML::Tiny
```
Comment (nyalldawson) [inline doc/linux.t2t]: Isn't it better to install this from the repos?

Comment (Gustry) [reply doc/linux.t2t]: I tried `sudo apt install libconfig-yaml-perl` but it was not working. Only this cpan command. Maybe another package was needed?

**Comment by jef-n** [comment]
superceeded by 6b6770816

**Comment by Gustry** [comment]
Thanks @jef-n

File: doc/linux.t2t (hunk 1 above)
Comment (3nids) [reply doc/linux.t2t]: It's libyaml-tiny-perl

**Pull request closed without merging**